openmc/examples.py

```diff
@@ -0,0 +1,619 @@
+import numpy as np
+
+import openmc
+import openmc.model
+
+
+def pwr_pin_cell():
+    """Create a PWR pin-cell model.
+
+    Returns
+    -------
+    model : openmc.model.Model
+        A PWR pin-cell model
+
+    """
+    model = openmc.model.Model()
+
+    # Define materials.
+    fuel = openmc.Material(name='Fuel')
+    fuel.set_density('g/cm3', 10.29769)
+    fuel.add_nuclide("U234", 4.4843e-6)
+    fuel.add_nuclide("U235", 5.5815e-4)
+    fuel.add_nuclide("U238", 2.2408e-2)
+    fuel.add_nuclide("O16", 4.5829e-2)
+
+    clad = openmc.Material(name='Cladding')
+    clad.set_density('g/cm3', 6.55)
+    clad.add_nuclide("Zr90", 2.1827e-2)
+    clad.add_nuclide("Zr91", 4.7600e-3)
+    clad.add_nuclide("Zr92", 7.2758e-3)
+    clad.add_nuclide("Zr94", 7.3734e-3)
+    clad.add_nuclide("Zr96", 1.1879e-3)
+
+    hot_water = openmc.Material(name='Hot borated water')
+    hot_water.set_density('g/cm3', 0.740582)
+    hot_water.add_nuclide("H1", 4.9457e-2)
+    hot_water.add_nuclide("O16", 2.4672e-2)
+    hot_water.add_nuclide("B10", 8.0042e-6)
+    hot_water.add_nuclide("B11", 3.2218e-5)
+    hot_water.add_s_alpha_beta('c_H_in_H2O')
+
+    # Define the materials file.
+    model.materials = (fuel, clad, hot_water)
```

Comment:
No, definitely not a hidden feature. If I change the examples to use it, it will change all the inputs_true.dat files. Instead, I described this in the docstring for `Model` so that users are aware.